collector/nats/examples/compose/README.md

```diff
@@ -0,0 +1,36 @@
+# Monitor NATS with the OpenTelemetry Collector for Lightstep
```

Comment:
> should we call "docker" example instead of "compose"?

Checkout https://github.com/compose-spec/compose-spec/blob/master/spec.md. Compose is more than a Docker thing now. Though only docker-compose implements all of the features. The Podman users will be delighted with the "compose" name.